api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -34,6 +34,8 @@
 	
 	public static final String DOWNLOAD_ATTACHMENT_URL = MODULE_BASE_URL + "/download";
 	
+	public static final String SEARCH_ATTACHMENT_URL = "/rest/v1/" + ATTACHMENT_URI;
```

Comment:
Please rename:
`SEARCH_ATTACHMENT_URL` → `ATTACHMENT_RESOURCE_URL`

Reply:
@mks-d I made a PR.Looking forward for the reviews. Thank you very much.